app/services/default_values_object.rb

```diff
@@ -0,0 +1,21 @@
+class DefaultValuesObject
```

Comment:
Even if there wasn't a method there for it already, defining something on the object you're acting on would be the better way.